meta/processor/src/test/java/net/skyscanner/backpack/ksp/ProcessorTests.kt

```diff
@@ -0,0 +1,252 @@
+/**
+ * Backpack for Android - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.skyscanner.backpack.ksp
+
+import org.junit.Test
+
+class ProcessorTests {
+
+    @Test
+    fun `view story generation`() = testKsp("""
+import net.skyscanner.backpack.meta.StoryKind
+
+@net.skyscanner.backpack.meta.ComponentMarker("Floating Action Button")
```

Comment:
The qualified names make the tests less bulky and more clear (shorter code)

Reply:
can you maybe extract the test files to variables, like input/output to make it easier to digest